debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerChangeTableSetIT.java

```diff
@@ -763,14 +763,26 @@ public void addDefaultValue() throws Exception {
         assertConnectorIsRunning();
         TestHelper.waitForSnapshotToBeCompleted();
 
+        TestHelper.waitForStreamingStarted();
+        TestHelper.waitForMaxLsnAvailable(connection);
+
         connection.execute("ALTER TABLE dbo.tableb ADD DEFAULT ('default_value') FOR colb");
         TestHelper.enableTableCdc(connection, "tableb", "after_change");
 
         connection.execute("INSERT INTO tableb VALUES('1', 'some_value')");
+        TestHelper.waitForCdcRecord(connection, "tableb", "after_change", rs -> rs.getInt("id") == 1);
+
         List<SourceRecord> records = consumeRecordsByTopic(1).recordsForTopic("server1.dbo.tableb");
         Assertions.assertThat(records).hasSize(1);
+        System.out.println("Records: " + records);
```

Comment:
We use Debug.println() or something like that elsewhere. Would be nice to align with that.

Reply:
Added a commit for this and applied. Thanks, @Naros!

Reply:
Damn, thanks for catching that.  Ya I forget about those debug classes and just go old school.  Thanks for fixing it tho!